packages/aws-client/src/DynamoDb.ts

```diff
@@ -0,0 +1,170 @@
+/**
+ * @module DynamoDb
+ */
+
+import { RecordDoesNotExist } from '@cumulus/errors';
+import { dynamodb } from './services';
+import { improveStackTrace } from './utils';
+
+/**
+ * Call DynamoDb client get
+ *
+ * See [DocumentClient.get()](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB/DocumentClient.html#get-property)
+ * for descriptions of `params` and the return data.
+ *
+ * @param {Object} params
+ * @param {string} params.tableName - Table name to read
+ * @param {any} params.item - Key identifying object to get
+ * @param {Object} params.client - Instance of a DynamoDb DocumentClient
```

Comment:
The type should be `AWS.DynamoDB.DocumentClient`?

Reply:
Fixed.